commit 4799f6856fdd38c8078a190eca3288029287cf66
Author: Ananth N Mavinakayanahalli <ananth@linux.vnet.ibm.com>
Date:   Tue Jul 17 11:32:37 2018 +0530

    MAINTAINERS: Add Naveen N. Rao as kprobes co-maintainer
    
    Naveen has been contributing consistently reviewing and hardening
    kprobes for some time now. I have not been able to do the same due
    to other commitments.
    
    Signed-off-by: Ananth N Mavinakayanahalli <ananth@linux.vnet.ibm.com>
    Cc: Naveen N. Rao <naveen.n.rao@linux.vnet.ibm.com>
    Cc: Linus Torvalds <torvalds@linux-foundation.org>
    Cc: Peter Zijlstra <peterz@infradead.org>
    Cc: Thomas Gleixner <tglx@linutronix.de>
    Cc: Anil S Keshavamurthy <anil.s.keshavamurthy@intel.com>
    Cc: "David S. Miller" <davem@davemloft.net>
    Cc: Masami Hiramatsu <mhiramat@kernel.org>
    Cc: Arnaldo Carvalho de Melo <acme@redhat.com>
    Cc: Namhyung Kim <namhyung@kernel.org>
    Cc: Jiri Olsa <jolsa@redhat.com>
    Cc: akpm@linux-foundation.org
    Cc: mhiramat@kernel.org
    Link: http://lkml.kernel.org/r/153180735790.1914.15547706781664285286.stgit@thinktux
    Signed-off-by: Ingo Molnar <mingo@kernel.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index 0fe4228f78cb..42a884c1b0f7 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7984,7 +7984,7 @@ F:	lib/test_kmod.c
 F:	tools/testing/selftests/kmod/
 
 KPROBES
-M:	Ananth N Mavinakayanahalli <ananth@linux.vnet.ibm.com>
+M:	Naveen N. Rao <naveen.n.rao@linux.vnet.ibm.com>
 M:	Anil S Keshavamurthy <anil.s.keshavamurthy@intel.com>
 M:	"David S. Miller" <davem@davemloft.net>
 M:	Masami Hiramatsu <mhiramat@kernel.org>